ui-manchette-with-spacetimechart/src/hooks/useManchetteWithSpaceTimeChart.ts

```diff
@@ -53,19 +51,13 @@ const useManchettesWithSpaceTimeChart = (
     scrollPosition: 0,
     isProportional: true,
     operationalPointsChart: [],
-    panY: false,
     scales: [],
   });
 
-  const { xZoom, yZoom, xOffset, yOffset, panY, panning, scrollPosition, isProportional } = state;
+  const { xZoom, yZoom, xOffset, yOffset, panning, scrollPosition, isProportional } = state;
 
   const paths = usePaths(projectPathTrainResult, selectedTrain);
 
-  const checkOverflow = useCallback((isOverflowFromCallback: boolean) => {
-    setState((prev) => ({ ...prev, panY: isOverflowFromCallback }));
-  }, []);
-  useIsOverflow(manchetteWithSpaceTimeChartContainer, checkOverflow);
-
   // Memoize timeWindow to avoid recalculation on each render
```

Comment:
```suggestion
  // Memorize timeWindow to avoid recalculation on each render
```
https://www.merriam-webster.com/dictionary/memorize

Reply:
"Memoize" is computer science wording:

- https://en.wikipedia.org/wiki/Memoization
- https://react.dev/reference/react/useMemo